ejerciciosSebaRata/matriz_lib.rb

```diff
@@ -0,0 +1,106 @@
+# Clase que contiene operacioness para realizarle a una matriz
+class MatrizLib
+
+    # inicializa nueva matriz 
+    # @params [Array<Array<Numeric>>] matriz de numeros
+    def initialize(matriz)
+        @matriz = matriz
+    end
+
+    # Saca los elementos de la diagonal principal
+    # @return [Array<Integer>] devuelve array con los elementos pertenecientes a la diag Principal
+    def sacar_elementos_diagonal
+        diagonal = []
+        @matriz.each_with_index do |fila, f|
+            elemento_diagonal = @matriz[f][f]
+            diagonal << elemento_diagonal
+        end
+        diagonal
+    end
+
+    #Valida si un elemento pertenece a la matriz en base a sus coordenadas
+    # @paramas [Integer] numeros enteros
+    # @return [Boolean] devuelve true si esta, false si no esta
+    def validar_coordenadas?(f, c)
+        f.between?(0, @matriz.size - 1) && c.between?(0, @matriz[f].size - 1)
+    end
+
+    #Saca los elementos adyacentes en base a un elemento
+    # @params [Integer] numeros enteros
+    # @return [Array <Numeric>] Array con los elementos adyacentes del elemento dado 
```

Comment:
Esto está bien correcto, así todos los métodos públicos tienen que estar documentados, los privados solo si creés que hace falta.
```suggestion
    # Saca los elementos adyacentes en base a un elemento
    # @params [Integer] numeros enteros
    # @return [Array <Numeric>] Array con los elementos adyacentes del elemento dado 
```